build/tools/avalanche-wallet-sdk/README.md

```diff
@@ -0,0 +1,125 @@
+
+# WARNING: Beta Release
```

Comment:
```suggestion
# 🔴WARNING: Beta Release🔴
```

Reply:
<img width="778" alt="Screen Shot 2021-06-28 at 1 58 24 PM" src="https://user-images.githubusercontent.com/12171681/123696393-f2d82b80-d818-11eb-93ab-d5940c2c24d3.png">

Reply:
We usually use
```
{% hint style="warning" %}
This is a warning!
{% endhint %}
```
Style hints to mark warnings like these. There is also "danger" and "info" styles for corresponding intensity.